package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "homebridge-jablotron",
-  "version": "2.5.7",
+  "version": "2.5.8",
```

Since this introduces new functionality let's bump the minor. 

```suggestion
  "version": "2.6.0",
```